internal/internal_task_handlers.go

```diff
@@ -988,7 +980,7 @@ ProcessEvents:
 			// workflow timeout.
 			return nil, nonDeterministicErr
 		default:
-			panic(fmt.Sprintf("unknown mismatched workflow history policy."))
+			panic("unknown mismatched workflow history policy.")
```

Comment:
we should really just check and fix all of these, bleh.  I thought I did that a while ago, but I might be thinking of some other codebase...

Reply:
I address things as I see in relevant places. Let's all encourage this and don't worry about having ~20% refactorings in a diff. More than that should probably be its own PR

Reply:
yea, I meant "fix them all" is probably a not-this-PR thing.  happy to approve obviously-safe changes like this as they appear 👍